wallpaper/wallpaper.go

```diff
@@ -0,0 +1,67 @@
+package wallpaper
+
+import (
+	"errors"
+
+	"github.com/godbus/dbus/v5"
+	"github.com/rymdport/portal/internal/apis"
+	"github.com/rymdport/portal/internal/convert"
+	"github.com/rymdport/portal/internal/request"
+)
+
+const (
+	interfaceName            = apis.CallBaseName + ".Wallpaper"
+	setWallpaperURICallName  = interfaceName + ".SetWallpaperURI"
+	setWallpaperFileCallName = interfaceName + ".SetWallpaperFile"
+)
+
+// WallpaperLocation is the type of the parameter SetOn of wallpaper options
+type WallpaperLocation uint
+
+const (
+	Background WallpaperLocation = iota // Set wallpaper of Background
+	Lockscreen                          // Set wallpaper of Locksreen
+	Both                                // Set wallpaper of both background and lockscreen
+)
```

Comment:
If you set the constants to the string values directly, you can remove the need for the map.